jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  coverageDirectory: 'coverage',
+  coverageProvider: 'v8',
+  coverageReporters: ['json', 'text'],
+  testTimeout: 20000,
```

Comment:
Bumped this up for the CI failures on macOS, but doesn't seem to really need it locally